src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -112,19 +112,26 @@ def two_groups_hyphens_two_replacements_with_suffix(
     "beautifulsoup4": ("bs4",),
     "bitvector": ("BitVector",),
     "cattrs": ("cattr", "cattrs"),
+    "django-activity-stream": ("actstream",),
     "django-cors-headers": ("corsheaders",),
     "django-countries": ("django_countries",),
     "django-filter": ("django_filters",),
     "django-fsm": ("django_fsm",),
+    "django-notifications-hq": ("notifications",),
+    "django-oauth-toolkit": ("oauth2_provider",),
     "django-object-actions": ("django_object_actions",),
+    "django-otp-yubikey": ("otp_yubikey",),
     "django-postgres-extra": ("psqlextra",),
     "django-redis": ("django_redis",),
     "django-scim2": ("django_scim",),
+    "django-two-factor-auth": ("two_factor",),
     "djangorestframework": ("rest_framework",),
     "djangorestframework-api-key": ("rest_framework_api_key",),
     "djangorestframework-dataclasses": ("rest_framework_dataclasses",),
+    "djangorestframework-jwt": ("rest_framework_jwt",),
     "djangorestframework-queryfields": ("drf_queryfields",),
     "djangorestframework-simplejwt": ("rest_framework_simplejwt",),
+    "drf-api-tracking": ("rest_framework_tracking",),    
```

Comment:
This should fix black+flake8
```suggestion
    "drf-api-tracking": ("rest_framework_tracking",),
```